### PR TITLE
DDS-1009 custom serialization via params

### PR DESCRIPTION
**Pull request by dmann**

### CODE REVIEW

File: app/api/dds/v1/children_api.rb [hunk 1]
```diff
@@ -14,6 +14,9 @@ class ChildrenAPI < Grape::API
       end
       params do
         optional :name_contains, type: String, desc: 'list children whose name contains this string'
+        optional :exclude_response_fields, type: Array[String],
```
Comment (dmlond) [inline app/api/dds/v1/children_api.rb:17]: you should run launch_application and view the /apiexplorer to see if/how this ends up in swagger.

File: spec/requests/children_api_spec.rb [hunk 2]
```diff
@@ -44,6 +44,20 @@
           deleted_folder,
           other_folder
         ] }
+
```
Comment (dmlond) [inline spec/requests/children_api_spec.rb:47]: can this be turned into a shared_example?

**Review by dmlond — CHANGES_REQUESTED** [review]
looks good, just a few suggestions

File: app/api/dds/v1/children_api.rb (hunk 1 above)
Comment (dmann) [reply app/api/dds/v1/children_api.rb:17]: Na, I don't feel like messing with `launch_application`. If after deploying to ua_test we find that it's not usable in swagger, I'll look into other solutions.

File: spec/requests/children_api_spec.rb (hunk 2 above)
Comment (dmann) [reply spec/requests/children_api_spec.rb:47]: Yeah, I was waffling between shared_context and shared_examples for this bit. I landed on not refactoring due to decision paralysis. If this feature sticks around and is expanded to other endpoints, I'll definitely refactor it and put it with the other request shared examples.

**Comment by dmann** [comment]
I will merge this into my main task branch and create a new PR to ua_test from there.

**Pull request closed without merging**